src/utils/secure.filter.ts

```diff
@@ -3,17 +3,29 @@ import { ApplicationData } from "../model";
 import { IsSecureRegisterKey, OverseasEntityKey, Transactionkey } from "../model/data.types.model";
 import { getApplicationData, setExtraData } from "./application.data";
 import { logger } from "./logger";
-import { getUrlWithTransactionIdAndSubmissionId, isRemoveJourney } from "../utils/url";
+import { getUrlWithTransactionIdAndSubmissionId, isRemoveJourney, isRegistrationJourney } from "../utils/url";
 import * as config from "../config";
 import { isActiveFeature } from "./feature.flag";
 import { updateOverseasEntity } from "../service/overseas.entities.service";
 import { Session } from "@companieshouse/node-session-handler";
 
+const fetchApplicationData = (req: Request, isRegistration: boolean): Promise<ApplicationData> => {
```

Comment:
Can't be sure from looking at the code in GitHub, but could you potentially just do the line:

`const isRegistration = isRegistrationJourney(req);
`
within this function and avoid the need for the second `isRegistration` parameter?

Reply:
Oh yes, thought about that, but...

1) We need that value elsewhere in the POST handler so makes sense running it once and passing it as a parameter, rather than running it twice.
2) It's temporary code that will be deleted once the Update portion of `secure-register-filter` is done so has only got a pretty short TTL.